cristin-organization/src/main/java/no/unit/nva/cristin/organization/CristinOrganizationApiClient.java

```diff
@@ -86,19 +89,24 @@ public Organization getOrganization(URI uri)
             throws NotFoundException, FailedHttpRequestException {
 
         URI queryUri = createCristinQueryUri(translateToCristinApi(requestQueryParams), UNITS_PATH);
-        final long start = System.currentTimeMillis();
         SearchResponse<Organization> searchResponse = query(queryUri);
+        final long start = System.currentTimeMillis();
```

Comment:
Why did you move this?

Reply:
A result of fat fingers and being asleep while undoing a refactor...